frontend/zc_messaging/src/pages/threads/threads.utils.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios"
+import { BASE_URL } from "../../utils/constants"
+
+/**
+ * @param {string} orgId
+ * @param {string} senderId
+ * @returns {Promise<Object>}
+ * @description Send get thread of all message from Database senderId is associated with
+ */
+
+const getThreadHandler = async (orgId, senderId) => {
+  try {
+    if (orgId && senderId) {
+      const getThreadRequest = await axios.get(
+        // supposed to be
+        // `${BASE_URL}/org/${orgId}/member/${senderId}/threads`
```

Comment:
switch if back to the base url and make this PR a draft until the update to the BE is ready

Reply:
The Backend Changes are ready Sir